<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-CJQ9RU0K" ignored="2" name="Default test(failed)" tests="3" failures="2" timestamp="2023-01-05T16:46:16 IST" time="26.383" errors="0">
  <testcase name="Test2" time="0.09" classname="FrameworkKekwords.FailedTest"/>
  <testcase name="Test5" time="0.032" classname="FrameworkKekwords.FailedTest">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at FrameworkKekwords.FailedTest.Test5(FailedTest.java:54)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- Test5 -->
  <testcase name="Test3" time="0.125" classname="FrameworkKekwords.FailedTest">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at FrameworkKekwords.FailedTest.Test3(FailedTest.java:42)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- Test3 -->
  <testcase name="Test1" time="0.0" classname="FrameworkKekwords.FailedTest">
    <ignored/>
  </testcase> <!-- Test1 -->
  <testcase name="Test4" time="0.0" classname="FrameworkKekwords.FailedTest">
    <ignored/>
  </testcase> <!-- Test4 -->
</testsuite> <!-- Default test(failed) -->
